packages/pyright-internal/src/tests/hoverProvider.test.ts

```diff
@@ -438,3 +438,133 @@ test('hover on __call__ method', async () => {
         marker2: '```python\n(variable) def foo(a: int) -> int\n```',
     });
 });
+
+test('hover on operators', async () => {
+    const code = `
+// @filename: test.py
+//// class A:
+////     pass
+//// class B:
+////     def __radd__(self, lhs: A) -> "B":
+////         return B()
+//// A() [|/*marker1*/+|] B()
+//// a = 1 [|/*marker2*/+|] 2
+//// a [|/*marker3*/+=|] 3
+//// b = ([|/*marker4*/~|]a [|/*marker5*/&|] 255,)
+//// c = 4 [|/*marker6*/*|] b
+//// d = 5 [|/*marker7*/not|] in c
```

Comment:
i think it makes more sense for the hover & definition to be on the `in` keyword instead of the `not` keyword (even though pylance only works on `not` for some reason?)

it looks like in your implementation it works on both, which i think is fine, but i think the test should be modified to ensure that it also works on the `in` keyword

Reply:
looks like the marker's been renamed but it still highlights `not` instead of `in`

<img width="571" height="220" alt="image" src="https://github.com/user-attachments/assets/ff272db5-e58d-477f-84cb-aea7bedd7fea" />

i think it should be:
```ts
//// d = 5 not [|/*marker6*/in|] c
```